src/libs/NetworkConnection.js

```diff
@@ -65,6 +74,10 @@ function subscribeToNetInfo() {
     // whether a user has internet connectivity or not.
     NetInfo.addEventListener((state) => {
         Log.info('[NetworkConnection] NetInfo state change', false, state);
+        if (shouldForceOffline) {
+            Log.info('[NetworkConnection] Not setting offline status because shouldForceOffline = true');
+            return;
+        }
```

Comment:
If you skip the network change event in this case, and then you flip the flip to force offline off, does the `offline status` end up in the right state?

For example:

1. You are truly offline
2. Set `shouldForceOffline` to `true` forcing offline
3. Go online (enable wifi or whatever)
4. Set `shouldForceOffline` to `false`

does the app think that we are offline or online?

Reply:
Tested this and it doesn't seem to be a problem